### PR TITLE
rebranded tea research international to marketing magnet and overhauled design

### PR DESCRIPTION
**Pull request by LemonFace0309**
redesigned homepaged to highlight the embedded features more

added link to docs and github code

updated integrations page

https://www.loom.com/share/bc2eacb093114b098ae6df2f2573fbf5

### CODE REVIEW

**Comment by LemonFace0309** [comment]
* **#35** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/35?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @LemonFace0309 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: components/LogIn.js
Comment (yeldah3) [inline components/LogIn.js:1]: Thanks for yeeting this component 🙌

File: components/Sidebar.js [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
+            <Item name="Dashboard" href="/" />
+            <Item name="Integrations" href="/integrations" />
+            <Item name="CRM" href="/integrations/export-crm" />
+            <Item name="Ad Platforms" href="/integrations/export-ads" />
+            <Item name="Import Dataset" href="/integrations/import-dataset" />
+          </nav>
+        </div>
+        <div className="hidden text-center text-lg italic text-stone-700 md:block md:text-2xl">
+          Superpowering marketing operations since{" "}
+          <abbr title="Did you know, marketing was first invented when someone figured out they could sell more woolly mammoth meat with a catchy slogan?">
```
Comment (yeldah3) [inline components/Sidebar.js:29]: 😂

File: components/SyncCreationWizard.js [hunk 2]
```diff
@@ -20,26 +21,28 @@ export default function SyncCreationWizard({
...
-            },
-          ])
-          await refetchSyncs()
-          // prepares a new link for the next sync creation
-          await resetSyncManagementLink()
-        }
-        setShowCreateSyncWizard(false)
-      }}
-    />
+    <Card>
```
Comment (yeldah3) [inline components/SyncCreationWizard.js:24]: dig this

File: .prettierrc.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
   "semi": false,
   "trailingComma": "all",
-  "printWidth": 110
+  "printWidth": 110,
+  "plugins": ["prettier-plugin-tailwindcss"]
```
Comment (yeldah3) [inline .prettierrc.json:5]: neat!

File: pages/index.js [hunk 4]
```diff
@@ -15,15 +15,14 @@ export default function Index() {
...
-        trends.
+        Unlock the full potential of your business with comprehensive data on retailers and trends.
       </p>
       <div className="grid grid-cols-1 gap-4 lg:grid-cols-2">
         <Card className="flex flex-col gap-3">
           <h3 className="font-medium text-teal-800">Retailer Datasets</h3>
           <ul className="ml-6 flex grow list-disc flex-col gap-1 text-sm text-stone-600">
             <li>
-              <Anchor href="/retailers">US tea boutiques</Anchor> updated 1 week ago
+              <Anchor href="/retailers">US boutiques</Anchor> updated 1 week ago
```
Comment (yeldah3) [inline pages/index.js:25]: Let's remove all these dead hrefs? They lead to 404s now!

File: pages/index.js
Comment (yeldah3) [inline pages/index.js:1]: Screenshot 2024-05-22 at 2 56 41 PM
Looks like the Trend Datasets card still references Tea rankings

**Review by yeldah3 — COMMENTED** [review]
Looking great!! Main blocker right now is there are a few links that lead to 404s on the dashboard page

**Review by yeldah3 — APPROVED** [review]

**Pull request merged**